client/src/panels/panel_declarations/services/services.scss

```diff
@@ -8,7 +8,7 @@
   }
 }
 .service-overview-rect {
-  background-color: lighten($tertiaryColor, 37%);
+  background-color: $backgroundColor;
```

Comment:
Original version should still use the tertiary color, your version should use white color